### PR TITLE
G20-554: Hypothesis client change language

### PR DESCRIPTION
**Pull request by mpereiraesaa**

### CODE REVIEW

File: src/sidebar/index.js [hunk 1]
```diff
@@ -101,8 +101,13 @@ function setupHttp($http, streamer) {
 }
 
 // @ngInject
-function setupI18n(i18nService) {
+function setupI18n($window, $rootScope, i18nService) {
   i18nService.initI18n();
+  angular.element($window).on('storage', function(event) {
+    if (event.key === 'locale') {
+      $rootScope.$apply();
```
Comment (jarlep) [inline src/sidebar/index.js]: What happens here?

Comment (jarlep) [reply src/sidebar/index.js]: Guess its fine https://github.com/angular/angular.js/wiki/Dev-Guide:-When-to-use-$scope.$apply()

Comment (jarlep) [reply src/sidebar/index.js]: Look forward to React version :-)

Comment (mpereiraesaa) [reply src/sidebar/index.js]: yei!

**Pull request merged**